### ISSUE TITLE
xyzloader not working with comma as separator

### ISSUE THREAD

**Issue by han16nah** [3dgeo-heidelberg/helios#62]
It seems to me like the xyzloader is not working when using comma as separator (or anything other than space?)

The survey aborts with an Exception:
```
Applying filter: xyzloader
Reading point cloud from XYZ file data/sceneparts/pointclouds/sphere_dens25000_sepcomma.xyz ...
Adding default material
XYZPointCloudFileLoader ERROR:
        Record/row with no (x,y,z) has been found.
Failed to read xyz point cloud file "data/sceneparts/pointclouds/sphere_dens25000_sepcomma.xyz"
EXCEPTION: Exception at XYZPointCloudFileLoader::firstPass
        Record with no (x,y,z)
```

Here is some sample [data.zip](https://github.com/3dgeo-heidelberg/helios/files/6954998/data.zip) (TLS sphere survey with the sphere point cloud in a comma-separated xyz file and `<param type="string" key="separator" value="," />` in the scene file).

**Comment by han16nah** [comment]
![grafik](https://user-images.githubusercontent.com/41050948/128719869-6798e586-e2a5-49f2-a6d7-b1baf341a15e.png)

@lwiniwar already helped me find the bug: We need a ! in front of the .empty() --> only overwrite the default separator (" ") if the parameter that is read from the XML file is NOT empty.

**Comment by han16nah** [comment]
fixed by #63
